frontend/src/scenes/sessions/SessionDetails.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import { Table } from 'antd'
+import { humanFriendlyDetailedTime } from '~/lib/utils'
+import { EventDetails, formatEventName } from 'scenes/events'
+import { Property } from 'lib/components/Property'
+
+export function SessionDetails({ events }) {
+    const columns = [
+        {
+            title: 'Event',
+            key: 'id',
+            render: function RenderEvent(event) {
+                return formatEventName(event)
+            },
+        },
+        {
+            title: 'URL / Screen',
+            key: 'url',
+            render: function renderURL(event) {
+                if (!event) return { props: { colSpan: 0 } }
+                let param = event.properties['$current_url'] ? '$current_url' : '$screen_name'
+                return <Property value={event.properties[param]} />
+            },
+            ellipsis: true,
+        },
+        {
+            title: 'Time',
+            render: function RenderEvent({ timestamp }) {
+                return <span>{humanFriendlyDetailedTime(timestamp)}</span>
+            },
+        },
+    ]
+
+    return (
+        <Table
+            columns={columns}
+            rowKey={event => event.id}
+            dataSource={events}
```

Comment:
Suggest adding `pagination={{ pageSize: 50, hideOnSinglePage: true }}` or something here, it's a bit annoying having to navigate through

Reply:
done!